content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md

```diff
@@ -0,0 +1,175 @@
+but we launched Kubernetes Upstream Training in Japan with the aim of increasing Japanese contributors by hosting a similar event in Japan.
+
+Before the pandemic, the training was held in person, but since 2020, it has been conducted online.
+The training offers the following content for those who have not yet contributed to Kubernetes:
+
+* Introduction to Kubernetes community
+* Overview of Kubernetes codebase and how to create your first PR
+* Tips and encouragement to lower participation barriers, such as language
+* How to set up the development environment
+* Hands-on session using [kubernetes-sigs/contributor-playground](https://github.com/kubernetes-sigs/contributor-playground)
+
+At the beginning of the program, we explain why contributing to Kubernetes is important and who can contribute.
+We emphasize that contributing to Kubernetes allows you to make a global impact and that Kubernetes community is looking forward to your contributions!
+
+We also explain Kubernetes community, SIGs, and Working Groups.
+Next, we explain the roles and responsibilities of Member, Reviewer, Approver, Tech Lead, and Chair.
+Additionally, we introduce the communication tools we primarily use, such as Slack, GitHub, and mailing lists.
+Some Japanese speakers may feel that communicating in English is a barrier.
+Additionally, those who are new to the community need to understand where and how communication takes place.
+We emphasize the importance of taking that first step, which is the most important aspect we focus on in our training!
+
+We then go over the structure of Kubernetes codebase, the main repositories, how to create a PR, and the CI/CD process using [Prow](https://docs.prow.k8s.io/).
+We explain in detail the process from creating a PR to getting it merged.
+
+After several lectures, participants get to experience hands-on work using [kubernetes-sigs/contributor-playground](https://github.com/kubernetes-sigs/contributor-playground), where they can create a simple PR.
+The goal is for participants to get a feel for the process of contributing to Kubernetes.
+
+At the end of the program, we also provide a detailed explanation of setting up the development environment for contributing to the `kubernetes/kubernetes` repository,
+including building code locally, running tests efficiently, and setting up clusters.
+
+## Interview with Participants
+
+We conducted interviews with those who participated in our training program.
+We asked them about their reasons for joining, their impressions, and their future goals.
+
+### [Keita Mochizuki](https://github.com/mochizuki875) ([NTT DATA Group Corporation](https://www.nttdata.com/global/en/about-us/profile))
+
+Keita Mochizuki is a contributor who continuously contributes to Kubernetes and related projects.
+Keita is also a professional in container security and has recently published a book.
```

Comment:
Nits: How about adding a link to the book here? 

@sftim - would that be considered as advertising and violating the guidelines?

Reply:
Yes, I _intentionally_ avoided this when I wrote this article — to ensure it wouldn’t be perceived as advertising or promotion.